examples/inverse/plot_lcmv_beamformer_volume.py

```diff
@@ -87,12 +88,20 @@
 
 clim = dict(kind='value', pos_lims=[0.3, 0.6, 0.9])
 stc.plot(src=forward['src'], subject='sample', subjects_dir=subjects_dir,
-         clim=clim)
+         clim=clim, verbose=True)
 
 ###############################################################################
-# We can also visualize the activity on a "glass brain" (shown here with
-# absolute values):
+# Now let's:
+# - visualize the activity on a "glass brain",
+# - show absolute values, and
+# - morph data to ``'fsaverage'`` instead of `sample`
 
+morph = mne.compute_source_morph(
+    forward['src'], 'sample', 'fsaverage', verbose=True)
 clim = dict(kind='value', lims=[0.3, 0.6, 0.9])
-abs(stc).plot(src=forward['src'], subject='sample', subjects_dir=subjects_dir,
-              mode='glass_brain', clim=clim)
+# XXX there is a bug with the max picking here, it's in a different place
+# (time course looks correct, but img does not)...
+# Also, clicking is buggy. This is in both glass_brain and stat_map modes.
+abs(stc).crop(0.08, 0.12).plot(
```

Comment:
uggh ... you are right, this is broken. I wonder if it has something to do with morphing.

Reply:
Yes perhaps, or something is wrong with the affine math. I'm going to diagnose by adding some `logger.debug` lines.